app/src/gui/layout/appBar.tsx

```diff
@@ -197,45 +251,64 @@ export default function MainAppBar(props: NavbarProps) {
     },
     projectList === null
       ? {
-          title: 'Loading notebooks...',
-          icon: <AccountTree />,
-          to: '/',
-          disabled: true,
-        }
+        title: 'Loading notebooks...',
+        icon: <DescriptionIcon />,
+        to: '/',
+        disabled: true,
+      }
       : isAuthenticated
         ? getNestedProjects(projectList)
         : {
-            title: 'Notebooks',
-            icon: <AccountTree />,
-            to: '/',
-            disabled: true,
-          },
-  ];
-  const bottomMenuItems: Array<MenuItemProps> = [
-    isAuthenticated
-      ? {
-          title: 'User',
-          icon: <AccountCircleIcon />,
-          to: ROUTES.SIGN_IN,
-          disabled: false,
-        }
-      : {
-          title: 'User',
-          icon: <AccountCircleIcon />,
+          title: 'Notebooks',
+          icon: <DescriptionIcon />,
           to: '/',
           disabled: true,
         },
+  ];
+
+  const bottomMenuItems: Array<MenuItemProps> = [
+
+    // @TODO Ranisa: Commented this for now, to be discussed if needed to display this nav item
+    // {
+    //   title: 'About Build',
+    //   icon: <SettingsIcon />,
+    //   to: ROUTES.ABOUT_BUILD,
+    //   disabled: false,
+    // },
+
     {
```

Comment:
We definitely need to keep this menu item, important for app management, dumping data and bug reporting.

Reply:
Noted